### PR TITLE
add countries endpoint

### PR DESCRIPTION
**Pull request by skynet67**

### CODE REVIEW

File: fitpay/src/main/java/com/fitpay/android/api/models/collection/ProvincesCollection.kt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.fitpay.android.api.models.collection
+
+import com.fitpay.android.api.models.Province
+
+/**
+ * Provinces list
+ */
+class ProvincesCollection(val iso: String,
```
Comment (jeremiahthomasharris) [inline fitpay/src/main/java/com/fitpay/android/api/models/collection/ProvincesCollection.kt]: Let's rename this ProvinceCollection for consistency

File: fitpay/src/main/java/com/fitpay/android/api/models/Country.kt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+              val names: Map<String, String>) : BaseModel() {
+
+    companion object {
+        private val PROVINCES = "provinces"
+    }
+
+    fun getProvinces(callback: ApiCallbackExt<ProvincesCollection>) {
+        makeGetCall<ProvincesCollection>(PROVINCES, null, ProvincesCollection::class.java, callback)
+    }
+}
```
Comment (jeremiahthomasharris) [inline fitpay/src/main/java/com/fitpay/android/api/models/Country.kt]: Add a `canGetProvinces()`

Comment (skynet67) [reply fitpay/src/main/java/com/fitpay/android/api/models/Country.kt]: done

File: fitpay/src/main/java/com/fitpay/android/api/models/collection/ProvincesCollection.kt (hunk 1 above)
Comment (skynet67) [reply fitpay/src/main/java/com/fitpay/android/api/models/collection/ProvincesCollection.kt]: done

**Pull request merged**